README.md

```diff
@@ -30,10 +30,12 @@ We're happy to review and approve new filters and actions that help you integrat
 * Change the least amount of code to achieve the goal; PRs with lots of whitespace changes and abstraction combined with a feature add or bug fix are difficult to review and may be rejected
 * Use the latest version of WordPress and turn `WP_DEBUG` on
 * If other plugins are installed during testing that might affect behavior, please list those in the PR description
-* Run the [Composer](https://getcomposer.org/doc/00-intro.md#installation-linux-unix-osx) code quality scripts on the files changed:
-    * `composer compat` to check for PHP version compatibility (currently 5.3), ignore files that you have not modified
-    * `composer phpcbf-path ./path/to/file/changed.php` to clean up whitespace
-    * `composer phpcs-path ./path/to/file/changed.php` to find other potential issues
+* Run the [Composer](https://getcomposer.org/doc/00-intro.md#installation-linux-unix-osx) scripts listed below before creating your PR:
```

Comment:
> Run... before creating your PR.

sounds like it's time to apply [this](https://blog.github.com/2012-09-17-contributing-guidelines/) and [this other](https://help.github.com/articles/about-issue-and-pull-request-templates/) blog posts.

Reply:
I totally agree. It's on our SDK quality list. I'll assign some tasks for that on the next sprint 👍